cmd/commands/git-source.go

```diff
@@ -697,6 +700,7 @@ func createGithubExampleIngress() *netv1.Ingress {
 				ServicePort: store.Get().GithubExampleEventSourceServicePort,
 			},
 		},
+		Annotations: map[string]string{"kubernetes.io/ingress.class": "nginx"},
```

Comment:
do we need that, when we have the default annotation on the IngressClass resource?
currently, our ingresses don't have anything "nginx" specific (even though I guess their entire setup is dependent upon nginx-style set up)

Reply:
It's a good comment , I don't want to have the default annotation on the ingress class.. on the other hand with this one we are only using nginx.. I'll check it @noam-codefresh